### PR TITLE
Removing the set_resolutions option to resolve mandatory receptor issues

### PR DESCRIPTION
**Pull request by itopcuoglu**

### CODE REVIEW

**Comment by psakievich** [comment]
Are you sure this is never needed?

**Comment by itopcuoglu** [comment]
None of the past regression tests use the `set_resolutions: no` option. We could also leave it in place, and make the code display a warning to say something like "WARNING: Nodal resolutions calculated by Tioga are not communicated at shared nodes. Use the default option "set_resolutions: yes" to ensure consistency at shared nodes".

**Comment by psakievich** [comment]
Seems reasonable to me then.

File: src/overset/TiogaOptions.C [hunk 1]
```diff
@@ -41,11 +38,6 @@ TiogaOptions::load(const YAML::Node& node)
   if (node["node_resolution_multiplier"]) {
     nodeResMult_ = node["node_resolution_multiplier"].as<double>();
   }
-
-  if (node["adjust_mandatory_fringe_resolutions"]) {
```
Comment (psakievich) [inline src/overset/TiogaOptions.C:45]: Should we delete the functions associated with this option as well?

Comment (itopcuoglu) [reply src/overset/TiogaOptions.C:45]: Yes, that would be more proper. Let me get that sorted out.

Comment (itopcuoglu) [reply src/overset/TiogaOptions.C:45]: I removed the boolean flag adjustResolutionsForFringes_, the function that checks it, and the flag checks themselves.

**Review by psakievich — APPROVED** [review]

**Pull request merged**